index.js

```diff
@@ -138,6 +138,18 @@ Hook.prototype.initialize = function initialize() {
     catch (e) { return this.log(this.format(Hook.log.binary, binary), 0); }
   }, this);
 
+  // in GUI clients node and npm are not in the PATH
+  // so get node binery PATH, add it to the PATH list
+  // and try again
+  if (!this.npm) {
```

Comment:
Ironically, pre-commit's own test suite doesn't work in such headless environments :) So I had to commit my changes from the CLI. Didn't have a chance to investigate what's the reason yet.